lib/arena/client.rb

```diff
@@ -43,18 +43,18 @@ def error?(response)
     end
 
     def error_message(response)
-      "#{response['status']['code']}: #{response['status']['message']} - #{response['status']['description']}"
+      "#{response['code']}: #{response['message']} - #{response['description']}"
```

Comment:
This looks wrong, but I should write a test for it. I had the wrong URL and it was failing on trying to get `[]` from `nil`.

Reply:
Yeah, errors return an object that has the keys `code`, `message`, `description`.